src/docs/metrics/etf-flow/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: ETF Flow Metrics
+author: Santiment Team
+date: 2025-03-28
+description: Inflow/Outflow from ETF providers
+---
+
+## Description
+
+Metrics that show stats related to the ETF inflow/outflow
+
+- `daily_etf_flow` - Daily ETF Flow per provider in USD
+- `total_etf_flow` - Total ETF Flow since inception per provider in USD
```

Comment:
I think we should expand a little bit more on the description. In its current form it requires you to know what inflow/outflow is, what `provider` is (is it just the `slug`?)

Reply:
the description is extended

yes, provider is slug